index.html

```diff
@@ -1,12 +1,48 @@
 <!DOCTYPE html>
 <html lang="en">
+
 <head>
     <meta charset="UTF-8">
     <meta http-equiv="X-UA-Compatible" content="IE=edge">
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
     <title>Document</title>
+    <link rel="stylesheet" href="style.css">
 </head>
+
 <body>
-    
+    <div class="container">
+        <nav class="mobile-nav">
+            <div class="shapes">
+                <div class="images">
+                    <img id="triangleImage" src="images/ShapeTriangle.png" alt="triangel shape">
+                    <img id="rectangleImage" src="images/Shape.png" alt="rectangle shape">
+
+                </div>
+            </div>
+            <div class="hamburger">
+                <span class="rec rectangle1"></span>
+                <span class="rec rectangle2"></span>
+                <span class="rec rectangle3"></span>
+            </div>
+        </nav>
+        <div class="main">
+            <div class="header">Ayesha Arshad</div>
+            <div class="discription">Hello! I am a software developer! I can help you build a product, feature or
+                website. Taka a look of my works.
+                If you like what you see and have a project you need coded, don’t hesitate and contact me.</div>
+            <div class="primary-button">
+                <button id="text" type="button">
+                    Start collaboration</button>
+            </div>
+            <div class="bg-image">
+                <img src="images/bg-cover-page.jpg" alt="background image for cover page">
+            </div>
+
+        </div>
+        <div class="footer">
+
+        </div>
+    </div>
```

Comment:
- [  ] Great job so far, But I think it would be better if you wrap the container in `section` tag because the Section tag defines the section of documents such as chapters, headers, footers, or any other sections. and it is important for the semantic of HTML. So please wrap it with a section instead of `div`.